CVLByExamples/BallGameExample/README.md

```diff
@@ -0,0 +1,32 @@
+# Ball Game
+
+This repository contains a simple example contract that shows how strengthening
+invariants is sometimes necessary to prove them.
+
+There are two versions of the contract:
+
+## Incorrect Code
+
+The version with a bug is `BallGameBroken.sol` in which the invariant is not preserved because the ball gets
+to player 4.
+
+The command for running this version:
+```certoraRun certora/specs/runBroken.conf```
+
+## Correct Code
+
+A correct contract is `BallGameCorrect.sol`, but even when trying to verify this contract with the previous spec `BallGame.spec` the invariant fails because the invariant is not strong enough.
+
```

Comment:
BallGame - there is no correct/incorrect code. There is one solidity file. and then one spec that is "correct" but not inductive and therefore can not be proved. and then there is the proven invariant